src/DDTrace/Integrations/Drupal/DrupalIntegration.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+namespace DDTrace\Integrations\Drupal;
+
+use DDTrace\GlobalTracer;
+use DDTrace\SpanData;
+use DDTrace\Integrations\Integration;
+use DDTrace\Tag;
+use DDTrace\Type;
+
+class DrupalIntegration extends Integration
+{
+    const NAME = 'drupal';
+
+    /**
+     * @return string The integration name.
+     */
+    public function getName()
+    {
+        return self::NAME;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function requiresExplicitTraceAnalyticsEnabling()
+    {
+        return false;
+    }
+
+    /**
+     * @return int
+     */
+    public function init()
+    {
+        if (!self::shouldLoad(self::NAME)) {
+            return Integration::NOT_LOADED;
+        }
+
+        $rootScope = GlobalTracer::get()->getRootScope();
+        $rootSpan = null;
+
+        if (null === $rootScope || null === ($rootSpan = $rootScope->getSpan())) {
+            return Integration::NOT_LOADED;
+        }
+
+        $integration = $this;
+
+        \DDTrace\trace_method(
+            'Drupal\Core\DrupalKernel',
+            'handle',
+            function (SpanData $span, $args) use ($rootSpan) {
+                if (!isset($args[0])) {
+                    return;
+                }
+
+                $req = $args[0];
+
+                $route = $req->getPathInfo();
+                $rootSpan->overwriteOperationName($route);
```

Comment:
Could these be ~uuids (raw paths) or are we able to obtain route/controller info here?  `http.url` is preferable to operation names for cardinality concerns and ~`drupal.handle` may be safer.  Also, where should be we setting `http.url`, `http.method`, `http.status_code`, etc?

Reply:
Updated - this uses the Drupal route where possible, e.g. previously the route was `/user/login`, with a Drupal route it is `user.login`. However the route information is not available all the time, so the operation name comes from Symfony instrumentation - `symfony.request`. Not ideal, but gets rid of the cardinality problem.